expat/fix-xmltest-log.sh

```diff
@@ -32,10 +32,10 @@ set -e
 
 filename="${1:-tests/xmltest.log}"
 
-dos2unix "${filename}"
-
-tempfile="$(mktemp)"
-sed \
+sed -i.bak \
+        -e '# convert DOS line endings without resorting to dos2unix' \
```

Comment:
Please mention the target of the conversion please — Unix line endings — not just the source, since conversion has two sides. Thanks!

Reply:
@dag-erling PS: yes, we have it in "dos2unix". To "convert DOS line endings" still feels off without saying where to. It's not a requirement to merge, but would be nice and symmetric, and not delay understanding to the end of the sentence.